callstats-java-sdk/src/main/java/io/callstats/sdk/httpclient/CallStatsAsyncHttpClient.java

```diff
@@ -42,19 +44,23 @@ public CallStatsAsyncHttpClient() {
 		Properties prop = new Properties();
 		InputStream input = null;
 	     
-		input = getClass().getClassLoader().getResourceAsStream(CallStatsConst.CallStatsJavaSDKPropertyFileName);
-		if (input !=null){
-			try {
+		//input = getClass().getClassLoader().getResourceAsStream(CallStatsConst.CallStatsJavaSDKPropertyFileName);
+		
+		try {
+			input = new FileInputStream(CallStatsConst.CallStatsJavaSDKPropertyFileName);
+			if (input !=null){
 				prop.load(input);
 				BASE_URL = prop.getProperty("CallStats.BaseURL");
 				connectionTimeOut = Integer.parseInt(prop.getProperty("CallStats.ConnectionTimeOut"));
```

Comment:
Good question, I think its better to have a default values for all the keys that are configurable through the config file. since i merged this PR will incorporate this in the next one.